src/components/dashboard/Rewards.js

```diff
@@ -35,18 +37,16 @@ const RewardsTab = props => {
     return `${Config.web3SiteUrl}/${path}?${query}`
   }
 
-  const getToken = async () => {
-    let token = (await userStorage.getProfileFieldValue('loginToken')) || ''
-    log.debug('got rewards login token', token)
-    setToken(token)
-  }
-
   useEffect(() => {
-    getToken()
+    (async () => {
```

Comment:
why? did you move the function inline?

Reply:
That function was being registered every time the component reloads and was only being used inside the effect which only runs once. I saw no point on that so moved it inside the effect

Reply:
but this is true for all functions used inside useEffect, if you are worried about performance then why not useCallback? defining a function and calling it is more verbose and readable